libraries/core_libs/consensus/src/vote_manager/vote_manager.cpp

```diff
@@ -227,9 +230,9 @@ bool VoteManager::addVerifiedVote(const std::shared_ptr<Vote>& vote) {
     LOG(log_nf_) << "Added verified vote: " << hash;
     LOG(log_dg_) << "Added verified vote: " << *vote;
 
-    // Save in db only those reward votes that have the same round as round during which we pushed the block into chain
-    if (is_valid_potential_reward_vote && reward_votes_round_ == vote->getRound()) {
```

Comment:
we dont need to check round anymore ?

Reply:
There is no harm to store reward votes from different rounds. Before we used it to enforce that the same round is used for all reward votes, but now this information is kept in 2t+1 votes so we use whatever round was in 2t+1 votes

Reply:
the harm is in performance that we save each such vote in db, even though we dont need it ?